packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -691,8 +692,14 @@ const initialize = async () => {
 	const state = props.node.state;
 	const modelId = props.node.inputs[0].value?.[0];
 	if (!modelId) return;
-	fetchConfigurations(modelId);
+	await fetchConfigurations(modelId);
+
 	model.value = await getModel(modelId);
+	if (suggestedConfigurationContext.value.tableData.length === 0 && model.value) {
+		await addDefaultConfiguration(model.value);
+		fetchConfigurations(modelId);
```

Comment:
could this be a then() call instead of an await?

Reply:
it could be, does the team like promise chaining better now? I like async/await

Reply:
It's not about liking one or the other. It's about, do we need to block the main thread for this?

Reply:
ahh i see, ya .then would work better here